app/views/projects/index.html.erb

```diff
@@ -12,25 +12,35 @@
 
 <div class="bg-white dark:bg-gray-800">
   <div class="flex flex-col">
-    <div class="overflow-x-auto">
+    <%= turbo_frame_tag "projects_list", src: projects_url(format: :turbo_stream, page: params[:page]), loading: :lazy do %>
       <table class="min-w-full table-fixed dark:divide-gray-600">
         <tbody
-          class="bg-white divide-y divide-gray-200  dark:bg-gray-800 dark:divide-gray-700"
+          class="
+            bg-white
+            divide-y
+            divide-gray-200
+            dark:bg-gray-800
+            dark:divide-gray-700
+          "
         >
-          <% @projects.each do |project| %>
-          <tr
-            class="text-sm font-normal text-gray-500  hover:bg-gray-50 dark:hover:bg-gray-700 dark:text-gray-400"
-          >
-            <td class="p-4 whitespace-nowrap">
-              <%= link_to project.human_name,
-                project_path(project),
-                class: "text-grey-900 dark:text-grey-100 font-semibold hover:underline" %>
-                <div class="text-sm font-normal text-gray-500 dark:text-gray-400"><%= project.description %></div>
+          <% 15.times do %>
```

Comment:
What is going on here exactly? Is this to display something like the skeleton loading in ant design?

Reply:
Sounds good!